docs/source/data/subscriptions.mdx

```diff
@@ -226,15 +314,15 @@ Use `withFilter` to make sure clients get exactly the subscription updates they
 
 ## Basic runnable example
 
-An example server is available on [GitHub](https://github.com/apollographql/docs-examples/blob/main/server-subscriptions/index.js) and CodeSandbox:
+An example server is available on [GitHub](https://github.com/apollographql/docs-examples/blob/main/server-subscriptions-as3/index.js) and CodeSandbox:
 
-<a href="https://codesandbox.io/s/github/apollographql/docs-examples/tree/main/server-subscriptions?fontsize=14&hidenavigation=1&theme=dark">
+<a href="https://codesandbox.io/s/github/apollographql/docs-examples/tree/main/server-subscriptions-as3?fontsize=14&hidenavigation=1&theme=dark">
   <img alt="Edit server-subscriptions" src="https://codesandbox.io/static/img/play-codesandbox.svg"></img>
 </a>
 
-The server exposes one subscription (`numberIncremented`) that returns an integer that's incremented on the server every second. The example requires only the `apollo-server` library.
+The server exposes one subscription (`numberIncremented`) that returns an integer that's incremented on the server every second.
 
-After you start up this server locally, you can visit `http://localhost:4000` to test out running the `numberIncremented` subscription in GraphQL Playground. You'll see the subscription's value update every second.
+After you start up this server locally, you can visit `http://localhost:4000` to test out running the `numberIncremented` subscription in Apollo Sandbox. You will most likely need to configure the subscriptions endpoint (ws://localhost:4000/graphql) as well in the settings menu. You'll see the subscription's value update every second.
```

Comment:
Should we suggest local or just assume using sandbox?

Also I think the visit link should be (a) a link not `` and (b) include `/graphql`

Reply:
What do you mean by suggest local?

Reply:
Er sorry. This PR says "After you start up this server locally" but there's a codesandbox link before it: maybe just document as if using codesandbox? Not sure if that's better.